taskcluster/ci/config.yml

```diff
@@ -0,0 +1,37 @@
+---
+trust-domain: mobile
+treeherder:
```

Comment:
Out of curiosity, since you mentioned you, once we have adhoc signing, do we intend to leave this support still happening or we can let this taskgraph do the job for us?

Reply:
Great question! 

I think adhoc signing is the ideal solution in the long run. That said, I expect FPM to die out before we have adhoc signing. FPM shouldn't be needed anymore after April. So, I'd say: if by any chance adhoc signing comes before, let's keep taskgraph this way. I don't think it's worth the effort to port this graph over.